crud/src/main/java/com/redhat/lightblue/rest/crud/health/CrudCheckConfiguration.java

```diff
@@ -87,12 +90,12 @@ private RolesProvider createRolesProvider() throws Exception {
 
         LdapConfiguration ldapConfiguration = new LdapConfiguration();
         ldapConfiguration.initializeFromJson(root);
-        
+
         return new CachedRolesProvider(new LdapRolesProvider(searchBase, ldapConfiguration),
```

Comment:
Yeah unfortunately the "login modules run in a separate classloader" issue is why this config file needs to exist (we tried without it in the beginning couldn't make it work).  Not sure we can share any initialization code, as the instance used by the login module is read from config parameters read from standalone.xml, not from the nice JSON file like we have.  App servers strike again